Makefile

```diff
@@ -428,3 +428,18 @@ kuttl-test-cleanup:
 	else \
 		echo "Namespce already cleaned up. Nothing to do"; \
 	fi
+
+# Used for webhook testing
+# The configure_local_webhook.sh script below will remove any OLM webhooks
+# for the operator and also scale its deployment replicas down to 0 so that
+# the operator can run locally.
+# We will attempt to catch SIGINT/SIGTERM and clean up the local webhooks,
+# but it may be necessary to manually run ./hack/clean_local_webhook.sh
+# before deploying with OLM again for other untrappable signals.
+SKIP_CERT ?=false
+.PHONY: run-with-webhook
+run-with-webhook: export METRICS_PORT?=8080
+run-with-webhook: export HEALTH_PORT?=8081
```

Comment:
as na fyi its good to use non default values here

i would recommend 

run-with-webhook: export METRICS_PORT?=33080
run-with-webhook: export HEALTH_PORT?=33081

the reason for that is so that we can run more then one operator locally with diffet terminals

otherwise you have to temperally modify the makefile or code.

can you also update the values here

https://github.com/openstack-k8s-operators/watcher-operator/blob/main/main.go#L64-L65

to use the same non default values so we can run it under vscode without having to configure the environment variable without colidiing with other operators.

Reply:
Done!